actionpack/test/controller/parameters/always_permitted_parameters_test.rb

```diff
@@ -20,7 +20,7 @@ def teardown
     end
   end
 
-  test "permits parameters that are whitelisted" do
+  test "permits parameters that are permitted" do
```

Comment:
I initially scrolled past this, but it does feel a bit tautological. What do you think of "allows both explicitly listed and always-permitted parameters"?